blenderless/scene.py

```diff
@@ -19,8 +19,8 @@ def import_bpy():
     The reason for this late-loading is that communication with Blender is
     restricted to a single separate render thread.
     """
-    os.environ["BLENDER_SYSTEM_SCRIPTS"] = 'external/blenderpy/bpy-2.91a0.data/scripts/2.91/scripts'
-    os.environ["BLENDER_SYSTEM_DATAFILES"] = 'external/blenderpy/bpy-2.91a0.data/scripts/2.91/datafiles'
+    os.environ["BLENDER_SYSTEM_SCRIPTS"] = 'external/bpy/2.92/scripts'
+    os.environ["BLENDER_SYSTEM_DATAFILES"] = 'external/bpy/2.92/datafiles'
```

Comment:
NOTE: looks like this is not needed any more.

Reply:
That would be great since this was quite a dirty fix.

Reply:
OK, I removed this.